AI-and-Analytics/End-to-end-Workloads/Census/README.md

```diff
@@ -74,23 +74,28 @@ To learn more about the extensions and how to configure the oneAPI environment,
 
 ### On Linux*
 
-1. Install the Intel® Distribution of Modin* python environment.
+1. Install the Intel® Distribution of Modin* python environment (Only python 3.8 - 3.10 are supported).
    ```
-   conda create -y -n intel-aikit-modin intel-aikit-modin -c intel
+   conda create -n modin-hdk python=3.x -y
```

Comment:
which package do you install for modin-hdk? I only saw python3.x installed

Reply:
sorry, only create a conda env named modin-hdk with python 3.x here, added modin-hdk installation in step 3